apps/linker/src/app/components/show-result/show-result.component.ts

```diff
@@ -0,0 +1,133 @@
+
+// eslint-disable-next-line
+declare let require: any;
+
+@Component({
+  selector: 'perun-web-apps-show-result',
+  templateUrl: './show-result.component.html',
+  styleUrls: ['./show-result.component.css'],
+})
+export class ShowResultComponent implements OnInit {
+  linkerResult: LinkerResult = 'UNKNOWN_ERROR';
+  loading = false;
+
+  constructor(
+    private route: ActivatedRoute,
+    private dialog: MatDialog,
+    private oauthService: OAuthService,
+    private attributeService: AttributesManagerService,
+    private storeService: StoreService,
+    private authzService: AuthzResolverService,
+    private rtMessages: RTMessagesManagerService,
+    private userFullNamePipe: UserFullNamePipe,
+    private oAuthStorage: OAuthStorage
+  ) {}
+
+  ngOnInit(): void {
+    this.route.params.subscribe((params) => {
+      this.linkerResult = params['result'] as LinkerResult;
+      (window.opener as Window).postMessage(this.linkerResult, '*');
+    });
+  }
+
+  closeWindow(): void {
+    window.close();
+  }
+
+  loginAgain(): void {
+    let queryParams = location.search.substring(1);
+    let index = 1;
+    if (this.linkerResult === 'IDENTITY_UNKNOWN') {
+      for (let i = 1; i < 10; i++) {
+        const formAccessToken = parseQueryParams('formerToken' + i.toString(), queryParams);
+        if (formAccessToken === undefined || formAccessToken == null || formAccessToken === '') {
+          index = i;
+          break;
+        }
+      }
+      queryParams =
+        queryParams + '&formerToken' + index.toString() + '=' + this.oauthService.getAccessToken();
+    }
+    sessionStorage.setItem('auth:queryParams', queryParams);
+    this.oauthService.initLoginFlow();
+  }
+
+  sendMessageToSupport(): void {
+    this.loading = true;
+    const currentUser = this.storeService.getPerunPrincipal().user;
+
+    const queryParams = location.search.substr(1);
+    this.oAuthStorage.setItem('access_token', parseQueryParams('formerToken', queryParams));
+    this.authzService.getPerunPrincipal().subscribe((principal) => {
+      const formerUser = principal.user;
+      this.rtMessages
+        .sentMessageToRTWithQueue(
+          'perun',
+          'Account linking: The accounts could not be automatically linked.',
+          this.getFullEmailBody(currentUser, formerUser)
+        )
+        .subscribe(
+          () => {
+            this.linkerResult = 'MESSAGE_SENT_TO_SUPPORT';
+            (window.opener as Window).postMessage(this.linkerResult, '*');
+            this.loading = false;
+          },
+          () => {
+            this.loading = false;
+            this.linkerResult = 'UNKNOWN_ERROR';
+            (window.opener as Window).postMessage(this.linkerResult, '*');
+          }
+        );
+    });
+  }
+
+  getFullEmailBody(currentUser: User, formerUser: User): string {
+    const instance = this.storeService.get('config') as string;
+    let text =
+      'Account linking: The accounts could not be automatically linked.\n' +
```

Comment:
Are we assuming that there will never be a requirement to localize these texts ?

Reply:
This text is sent to user support- so that would be kind of sad if they don't speak english